src/web/mjs/HakuNeko.mjs

```diff
@@ -79,6 +81,10 @@ export default class HakuNeko {
         return this._enums;
     }
 
+    get I18n() {
```

Comment:
A mechanic for localization on a global level should be done in a separate PR and from my point of view the current version of HakuNeko is very unlikely to implement localization at all.

If the new frontend supports localization, i recommend a scoped solution only for the new frontend.